services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/modules/db/repositories/resource_tracker.py

```diff
@@ -11,7 +12,16 @@
 
 
 class ResourceTrackerRepository(BaseRepository):
-    async def upsert_resource_tracker_container_data_(
+    async def get_prometheus_last_scraped_timestamp(self) -> datetime | None:
+        async with self.db_engine.begin() as conn:
```

Comment:
question regarding sqlalchemy 2.0. I know they do not anymore do the autocommit thing but do you always need to begin a transaction or could you in case of select just do
```python
async with self.db_engine.connect() as conn:
    ...
```
as well?

Reply:
I guess yes! but i did not explicitly install sqlalchemy, I have it as a transitive dependency from some of our packages, which are still at 1.4.48

Reply:
but I will modify it in the upcoming PR :+1: